HighwayPatrol.cpp

```diff
@@ -26,8 +28,14 @@ void HighwayPatrol::pullOver( Vehicle* v, bool willArrest, Highway* h )
     std::cout << name << ": vehicle is traveling " << v->speed - h->speedLimit << " miles per hour over the speed limit" << std::endl;
     if( willArrest )
     {
-        //print the vehicle type in this std::cout between "THE [" and "] PULL". 
-        std::cout << name << ": YOU IN THE [ " << " ] PULL OVER AND SHOW YOUR HANDS" << std::endl;
+        //print the vehicle type in this std::cout between "THE [" and "] PULL".
+        std::string vehicleType { };
+        
+        if (dynamic_cast<Car*>(v)) { vehicleType = "CAR"; }
+        if (dynamic_cast<Motorcycle*>(v)) { vehicleType = "MOTORCYCLE"; }
+        if (dynamic_cast<SemiTruck*>(v)) { vehicleType = "SEMITRUCK"; }
```

Comment:
- [ ] use `else if`.  Once the cast succeeds, stop trying to cast to the other types.